frontend/src/components/common/form/Text.tsx

```diff
@@ -0,0 +1,23 @@
+import { getIn, useFormikContext } from 'formik';
+import React from 'react';
+import styled from 'styled-components';
+
+export type TextProps = {
+  /** The field name */
+  field?: string;
+  /** Adds a custom class to the span element of the <Text> component */
+  className?: string;
+};
+
+export const Text: React.FC<TextProps> = ({ field, className, children }) => {
+  // if "field" prop is supplied, we render text content from formik field
+  // if no field is supplied, then we render the children as-is
+  const { values } = useFormikContext();
+  const value = field ? getIn(values, field) : children;
+
+  return <StyledSpan className={className}>{value}</StyledSpan>;
+};
+
+const StyledSpan = styled.span`
+  color: ${props => props.theme.css.formControlTextColor};
+`;
```

Comment:
New component to support read-only forms. Avoids having to re-skin inputs etc without borders.

Reply:
Disabled only apply to input fields. This component renders text as a span